tests/sles4sap/trento/install_agents.pm

```diff
@@ -31,7 +31,7 @@ sub run {
         }
     }
 
-    $cmd = $self->install_agent($wd, '/root/test', $agent_api_key, '10.0.0.4');
+    $cmd = trento::install_agent($wd, '/root/test', $agent_api_key, '10.0.0.4');
```

Comment:
IP is hard coded for which reason?

Reply:
I do not have a way to dynamically calculate it yet, TODO